readers/api/endpoint_test.go

```diff
@@ -618,8 +648,9 @@ func TestReadAll(t *testing.T) {
 			token:  userToken,
 			status: http.StatusOK,
 			res: pageRes{
-				Total:    uint64(len(queryMsgs)),
-				Messages: queryMsgs[0:10],
+				PageMetadata: readers.PageMetadata{Limit: 10, Format: "messages", Publisher: pubID2},
```

Comment:
```suggestion
				PageMetadata: readers.PageMetadata{Limit: 10, Publisher: pubID2},
```
Since the default format is messages no need to add it. Same as other cases

Reply:
Leaving it out returns an empty format field
![image](https://github.com/absmach/magistrala/assets/44696487/17313670-bddb-4b8a-b63e-5123c873c082)